spec/array_database_spec.rb

```diff
@@ -24,19 +24,19 @@
     end
   end
 
-  describe '#any?' do
-    it 'returns true if array database has any contacts' do
+  describe '#no_contacts?' do
```

Comment:
There is this unspoken rule that boolean methods read better when they are positive, as in `ready?` `empty?` etc. but not `not_ready?` or `not_empty?` because it's a bit of a mind twister to say:

"`no_contacts?` is false if there are contacts" ([Think of our poor friend Boole](http://sydneypadua.com/2dgoggles/happy-200th-birthday-george-boole/)).


------
![boole](https://user-images.githubusercontent.com/2157089/96930125-1c36ff80-14b3-11eb-83de-c7faf9ee8a3e.jpg)